### ISSUE TITLE
User account for vmware_exporter

### ISSUE THREAD

**Issue by cyanide-16** [pryorda/vmware_exporter#349]
Hi,

Could we use a separate user account to scrape these metrics instead of administrator. If yes, please let me know how to create one?

TIA

**Comment by lethargosapatheia** [comment]
Yes, you can, but this is related to vsphere's permissions, not to vmware_exporter. I don't remember the details, but if you read a little bit about it, you'll understand it eventually. You could start here: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html
